data_pipeline/extract_history.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import json
+import pandas as pd
+from collections import defaultdict
+from pandas import isnull
+from pandas.io.json import json_normalize
+import itertools
+import yaml
```

Comment:
if we are using pyyaml, can we add that to the Pipfile? We can merge this and I can add this after the fact, just to make it easier to run this script elsewhere.

Reply:
done